New package: SharedPreferences v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SharedPreferences
- Repository: https://github.com/raphasampaio/SharedPreferences.jl
- Created by: @raphasampaio
- Version: v0.1.0
- Commit: e8166682fb0e6777e5c551d3eb0eac92a4d0b605
- Reviewed by: @raphasampaio
- Reference: https://github.com/raphasampaio/SharedPreferences.jl/commit/e8166682fb0e6777e5c551d3eb0eac92a4d0b605#commitcomment-144409345
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62b805b67854e0d18f174905b376973c15551db5d8bfb2f15d9adc897d690fa8c1815c21bd47772eafacf32c8f8a7c819d96e21dc6680b07b7c827488df3e381686e48a2f5eb4a9ccf3b17a8a0309119f7bc9a0a7b3fad7489b61fd8424dd03402bba81a2384f941f9ca7539cbb4591aa93dbcb4f6dff681f5c9f5eee970f84bd1e935677a1c47b27d7063f5729a44acf3a2776b98e3b550043b2720c0bc35a542d9aab4120d17fd2fe02084fc8b289445bb12c5b136cabf2d00e2f559ef67ba6 -->

goerz commented:
[noblock] The badges in the README that should be linking to the documentation are currently giving a 404. I'm assuming these will start to work once this merge goes through and there's an actual release, but please make sure to check up on that.

JuliaRegistrator committed:
New package: SharedPreferences v0.1.0

UUID: a5a92e11-d0bb-45ff-9fc3-c070cd77387a
Repo: https://github.com/raphasampaio/SharedPreferences.jl.git
Tree: 5346b3c97a1442934cad6899d49cde9548a6802e

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
